### DISCUSSION TITLE
Join Discord

### DISCUSSION THREAD

**Discussion by ashramwen** [tailwindcss-tw/tailwindcss.com#63] [General]
https://discord.gg/4VwJ67hz4j

**Comment by beareatapple** [comment]
我被禁言了啦 QQ

**Reply by beareatapple** [reply]
@YC999333 @yphsiang @huibizhang 哇哇哇～～～

**Reply by huibizhang** [reply]
我知道，我都知道。

**Reply by huibizhang** [reply]
@beareatapple 看臉書

**Reply by beareatapple** [reply]
https://www.facebook.com/photo?fbid=122151313397586&set=a.122151406730910 那隻兔子是假的！
@YC999333 @yphsiang @ashramwen
